package.json

```diff
@@ -1,27 +1,28 @@
 {
   "name": "resin-semver",
   "version": "0.1.1",
-  "description": "resin specific semver utility methods",
+  "description": "Resin.io specific semver utility methods",
   "main": "src/index.js",
   "typings": "src/index.d.ts",
   "scripts": {
     "test": "mocha -r ts-node/register --outDir tmp test/index.spec.ts",
-    "build": "tsc src/index.ts"
+    "build": "tsc src/index.ts",
+    "ci": "npm run test"
```

Comment:
Don't think you need this - if you drop this script, and don't specify a `script` in `.travis.yml` at all then it'll run `npm test` [by default](https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Default-Test-Script) anyway.